groups-react/README.md

```diff
@@ -0,0 +1,12 @@
+# io.Connect Desktop Groups in React
+
+This is a template for creating a React Groups application for [**io.Connect Desktop**](https://interop.io//).
```

Comment:
Template for creating a \[custom Web Group App](https://docs.interop.io/desktop/capabilities/windows/window-management/overview/index.html#extending_web_groups) for \*\*io.Connect Desktop\*\*.